entproto/extension.go

```diff
@@ -22,12 +22,70 @@ import (
 	"path/filepath"
 	"strings"
 
+	"entgo.io/ent/entc"
 	"entgo.io/ent/entc/gen"
 	"github.com/jhump/protoreflect/desc"
 	"github.com/jhump/protoreflect/desc/protoprint"
 	"go.uber.org/multierr"
 )
 
+// ExtensionOpt is an option for the entproto extension.
+type ExtensionOpt func(*Extension)
```

Comment:
Let's keep the naming the same as entgql? https://github.com/ent/contrib/blob/master/entgql/extension.go#L41
```suggestion
type ExtensionOption func(*Extension)
```